### ISSUE TITLE
Push task broken in 2.8.0, 2.8.1

### ISSUE THREAD

**Issue by alandipert** [boot-clj/boot#705]
### Problem Description

The `push` task doesn't work because pomegranate now requires a checksum in order to push:

```
Sending maven-metadata.xml to https://repo.clojars.org/ (2k)
Could not transfer metadata adzerk:boot-reload/maven-metadata.xml from/to clojars (https://repo.clojars.org/): Access denied to: https://repo.clojars.org/adzerk/boot-reload/maven-metadata.xml, ReasonPhrase: Forbidden - no checksums provided for boot-reload-0.6.0.jar.asc.
```

(reported in Slack by @Deraen)

### Steps to reproduce

1. Attempt to push an artifact to Clojars with `push`

**Comment by alandipert** [comment]
The change in default that causes this is here: https://github.com/cemerick/pomegranate/pull/83/commits/373376252f427484ea753eb112852e32facb7217

A workaround might be to manually set the repo option from `:fail` to `:warn` when passing `push` the `repo-map` argument.

**Comment by alandipert** [comment]
Closed by #708
